ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -970,109 +969,28 @@ def test023_create_existing_order_with_test_units_and_change_article(self):
-        self.base_selenium.LOGGER.info('Press archive on suborder no #{}'.format(suborder_index_to_archive + 1))
-        self.order_page.archive_suborder(index=suborder_index_to_archive, check_pop_up=False)
-        rows_count = self.base_selenium.get_table_rows(element='order:suborder_table')
-
-        self.base_selenium.LOGGER.info('+ Number of suborders = {}'.format(len(rows_count)))
-        self.base_selenium.LOGGER.info(
-            '+ Assert  before clicking archive suborders count {}, after clicking archive suborders count {}'.format(3,
-                                                                                                                     len(
-                                                                                                                         rows_count)))
-        self.assertEqual(len(rows_count), 3)
-
-        # self.base_selenium.refresh()
-        order_url = self.base_selenium.get_url()
-        self.base_selenium.get(
-            url=order_url, sleep=self.base_selenium.TIME_MEDIUM)
-
-        self.order_page.archive_suborder(index=suborder_index_to_archive, check_pop_up=True)
-
-        self.base_selenium.get(
-            url=order_url, sleep=self.base_selenium.TIME_MEDIUM)
-        self.order_page.get_suborder_table()
-        rows_count = self.base_selenium.get_table_rows(element='order:suborder_table')
-
-        self.base_selenium.LOGGER.info('+ Number of suborders = {}'.format(len(rows_count)))
-        self.base_selenium.LOGGER.info(
-            '+ Assert  before archiving suborders count {}, after archiving suborders count {}'.format(3,
-                                                                                                       len(rows_count)))
-        self.assertEqual(len(rows_count), 2)
-
-        self.order_page.get_orders_page()
-
-        # going to archived orders page to make sure that the order is archived
-        self.order_page.get_archived_items()
-
-        self.order_page.open_filter_menu()
-        self.orders_page.filter_by_analysis_number(filter_text=analysis_no)
-
-        rows_count = self.order_page.result_table()
-        self.base_selenium.LOGGER.info(
-            '+ Assert count archived orders with a specific analysis number: {}'.format(len(rows_count)))
+        LIMS-6518
+        """
+        self.base_selenium.LOGGER.info('select random record')
+        orders, payload = self.orders_api.get_all_orders()
+        order_data = random.choice(orders['orders'])
+
+        order_no = order_data['orderNo']
+        self.order_page.apply_filter_scenario(filter_element='orders:filter_order_no', filter_text=order_no, field_type='text')
+        suborders_data = self.order_page.get_child_table_data(index=0)
+        self.order_page.archive_table_suborder(index=0)
+        self.base_selenium.LOGGER.info('make sure that suborder is archived successfully')
+        if len(suborders_data) > 1:
```

Comment:
to make sure that suborder is archived successfully, you should assert it's in archived table or to search for it in analysis active table

Reply:
Done